classes/class-rest-mappings-controller.php

```diff
@@ -225,16 +267,7 @@ public function prepare_item_for_response( $item, $request ) {
 			return new WP_REST_Response( $item );
 		}
 
-		$data = array(
-			'id'     => absint( $item->get_id() ),
-			'domain' => $item->get_domain(),
-			'active' => $item->is_active(),
-		);
-
-		// Return blog ID if sent
-		if ( null !== $request->get_param( 'blog' ) ) {
-			$data['blog'] = $request->get_param( 'blog' );
-		}
+		$data = $this->mapping_to_array( $item, $request );
```

Comment:
Any reason for this move? You could just return the data from `prepare_item_for_response` rather than a response object, it's a little more idiomatic with the standard name.